src/Meshes/abstractMesh.ts

```diff
@@ -1305,11 +1306,19 @@ export class AbstractMesh extends TransformNode implements IDisposable, ICullabl
 
     /** @hidden */
     public _getPositionData(applySkeleton: boolean, applyMorph: boolean): Nullable<FloatArray> {
-        var data = this.getVerticesData(VertexBuffer.PositionKind);
+        let needUpdatePositionInSkeletonPass = false;
+        let data = this.getVerticesData(VertexBuffer.PositionKind);
 
         if (data && ((applySkeleton && this.skeleton) || (applyMorph && this.morphTargetManager))) {
             data = Tools.Slice(data);
             this._generatePointsArray();
+            needUpdatePositionInSkeletonPass = !!(applySkeleton && this.skeleton) && !(applyMorph && this.morphTargetManager);
+
+            if (this._positions) {
+                this._internalAbstractMeshDataInfo._positions = this._positions.slice();
```

Comment:
Maybe use `Tools.Slice(this._positions)` instead (as above line 1313)?

Reply:
Good catch!